packages/@webex/plugin-cc/src/services/WebCallingService.ts

```diff
@@ -65,4 +63,51 @@ export default class WebCallingService {
   public async deregisterWebCallingLine() {
     this.line?.deregister();
   }
+
+  public answerCall(localAudioStream: LocalMicrophoneStream, taskId: string) {
+    if (this.call) {
+      try {
+        this.webex.logger.info(`[WebRtc]: Call answered: ${taskId}`);
```

Comment:
Any reason why we want to do [WebRTC] when this isn't the pattern elsewhere?

Reply:
Just followed the same pattern as agent desktop

Reply:
Removed this log now